### PR TITLE
hardening/807_add_admin_log_route

### PR DESCRIPTION
**Pull request by pcoello25**
* Partially fix issue #807 
  * Update CHANGES_NEXT_RELEASE is neccesary (first time).
* 100% unit tests passed:
```
Results : Tests run: 66, Failures: 0, Errors: 0, Skipped: 0
```
* e2e (unofficial) tests passed:
```
$ curl -X GET "http://localhost:8081/admin/log"
{"level": "DEBUG"}

$ curl -X GET "http://localhost:8081/admin/log?verbose=false"
{"level": "DEBUG"}

$ curl -X GET "http://localhost:8081/admin/log?verbose=true"
{"success":"true","log4j":{"level":"DEBUG","appenders":[{"name":"console","layout":"time=%d{yyyy-MM-dd}T%d{HH:mm:ss.SSSzzz} | lvl=%p | corr=%X{correlatorId} | trans=%X{transactionId} | srv=%X{service} | subsrv=%X{subservice} | function=%M | comp=Cygnus | msg=%C[%L] : %m%n"}]}}

[USING JSON TOOL]
$ curl -X GET "http://localhost:8081/admin/log?verbose=true" | python -m json.tool
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   278  100   278    0     0  46888      0 --:--:-- --:--:-- --:--:-- 55600
{
    "log4j": {
        "appenders": [
            {
                "layout": "time=%d{yyyy-MM-dd}T%d{HH:mm:ss.SSSzzz} | lvl=%p | corr=%X{correlatorId} | trans=%X{transactionId} | srv=%X{service} | subsrv=%X{subservice} | function=%M | comp=Cygnus | msg=%C[%L] : %m%n",
                "name": "console"
            }
        ],
        "level": "DEBUG"
    },
    "success": "true"
}
```
* Assignee: @frbattid

### CODE REVIEW

**Comment by frbattid** [comment]
The documentation is missing. Please, add it.

**Comment by pcoello25** [comment]
I have just add the documentation required. Forgot to create it!

File: doc/cygnus-common/installation_and_administration_guide/management_interface.md [hunk 1]
```diff
@@ -242,30 +242,43 @@ Response:
 [Top](#top)
 
 ##<a name="section9"></a>`GET /admin/log`
-Gets the log4j configuration (relevant parts, as the logging level or the appender names and layouts).
-
+Gets the logging level.
 ```
-GET http://<cygnus_host>:<management_port>/admin/log
+GET http://<cygnus_host>:<management_port>/admin/log4j
```
Comment (frbattid) [inline doc/cygnus-common/installation_and_administration_guide/management_interface.md]: log4j --> log

File: doc/cygnus-common/installation_and_administration_guide/management_interface.md [hunk 2]
```diff
@@ -242,30 +242,43 @@ Response:
 [Top](#top)
 
 ##<a name="section9"></a>`GET /admin/log`
-Gets the log4j configuration (relevant parts, as the logging level or the appender names and layouts).
-
+Gets the logging level.
```
Comment (frbattid) [inline doc/cygnus-common/installation_and_administration_guide/management_interface.md]: "If not parameterized with `verbose`, it simply gets the logging level".

File: doc/cygnus-common/installation_and_administration_guide/management_interface.md [hunk 3]
```diff
@@ -242,30 +242,43 @@ Response:
...
-            }
-        ],
-        "level": "..."
-    },
-    "success": "true"
+    "error": "..."
 }
 ```
 
+Gets the log4j configuration (relevant parts, as the logging level or the appender names and layouts) given a parameter `verbose` with a true value. `false` value is invalid.
```
Comment (frbattid) [inline doc/cygnus-common/installation_and_administration_guide/management_interface.md]: Instead, if parameterized with `verbose=true` gets the log4j configuration (relevant parts, as the logging level or the appender names and layouts).

Comment (pcoello25) [inline doc/cygnus-common/installation_and_administration_guide/management_interface.md]: Fixed in  832bedc

File: doc/cygnus-common/installation_and_administration_guide/management_interface.md (hunk 2 above)
Comment (pcoello25) [inline doc/cygnus-common/installation_and_administration_guide/management_interface.md]: Fixed in  832bedc

File: doc/cygnus-common/installation_and_administration_guide/management_interface.md (hunk 1 above)
Comment (pcoello25) [inline doc/cygnus-common/installation_and_administration_guide/management_interface.md]: Fixed in  832bedc

**Comment by pcoello25** [comment]
I've change the behaviour of the method. `verbose=false` now do the same as get method without parameter. Only show more information with `verbose=true`.

File: doc/cygnus-common/installation_and_administration_guide/management_interface.md [hunk 4]
```diff
@@ -242,8 +242,7 @@ Response:
 [Top](#top)
 
 ##<a name="section9"></a>`GET /admin/log`
-Gets the log4j configuration (relevant parts, as the logging level or the appender names and layouts).
-
+If parameterized with `verbose=true` (or directly, the `verbose` parameter is avoided), it simply gets the logging level.
```
Comment (frbattid) [inline doc/cygnus-common/installation_and_administration_guide/management_interface.md]: verbose=false

Comment (pcoello25) [inline doc/cygnus-common/installation_and_administration_guide/management_interface.md]: Fixed in  b9dd055

**Comment by pcoello25** [comment]
`Apiary.apib` modified in order to update the information with these api actions.

File: doc/apiary/apiary.apib [hunk 5]
```diff
@@ -601,6 +601,39 @@ Error list (HTTP response code in parenthesis):
 
 ## Group Admin API
 
+### Gets the log4j logging level [GET /admin/log]
+
++ Parameters
+
+       + ngsi_version: true or false (this case is equals to avoid the parameter)
```
Comment (frbattid) [inline doc/apiary/apiary.apib]: (this case is the same than avoiding the `verbose` parameter, see next section)

Comment (pcoello25) [inline doc/apiary/apiary.apib]: Fixed in  2be4a30

**Comment by frbattid** [comment]
LGTM

**Pull request merged**